src/main/java/com/onarandombox/MultiverseCore/commands/VersionCommand.java

```diff
@@ -44,9 +44,10 @@
     public VersionCommand(MultiverseCore plugin) {
         super(plugin);
         this.setName("Multiverse Version");
-        this.setCommandUsage("/mv version " + ChatColor.GOLD + "-[bhp] [--include-plugin-list]");
+        this.setCommandUsage("/mv version " + ChatColor.GOLD + "[-b|-h|-p] [--include-plugin-list]");
```

Comment:
This is the standard we use elsewhere?

Reply:
I think so.

https://github.com/Multiverse/Multiverse-Core/blob/master/src/main/java/com/onarandombox/MultiverseCore/commands/CreateCommand.java#L34

https://github.com/Multiverse/Multiverse-Core/blob/master/src/main/java/com/onarandombox/MultiverseCore/commands/DebugCommand.java#L26

https://github.com/Multiverse/Multiverse-Core/blob/master/src/main/java/com/onarandombox/MultiverseCore/commands/ModifyCommand.java#L30

Reply:
Most other command had the `-` inside the `[]` (so like `[-s]`, tho this is the only instance there is multiple of them. I thought this looks clearer, bcu `-[bph]` could be misunderstood as doing `-bhp` and not `-b` or `-h` or `-p